src/coreclr/vm/peassembly.cpp

```diff
@@ -260,7 +266,11 @@ TADDR PEAssembly::GetIL(RVA il)
     CONTRACT_END;
 
     PEImageLayout *image = NULL;
+#ifdef PEIMAGE_FLAT_LAYOUT_ONLY
```

Comment:
It may be better to return the flat layout from `GetLoadedLayout`. I suspect that we would need to ifdef a lot of calls to `GetLoadedLayout` otherwise.

Reply:
It sounds reasonable to. It would be best to do that in a separate PR.